### ISSUE TITLE
prysm on goerli cannot find suitable peers

### ISSUE THREAD

**Issue by joeytwiddle** [eth-educators/eth-docker#785]
I am trying to use goerli network with prysm (alongside geth).

The execution client has synced fine, but the consensus client has not. It shows:

```
level=info msg="Waiting for enough suitable peers before syncing" prefix=initial-sync required=3 suitable=0
level=info msg="Peer summary" activePeers=3 inbound=1 outbound=2 prefix=p2p
```

I have opened firewall port 9000 on TCP and UDP.

By the way, during config I:
- Disabled CI Rapid Sync
- Disabled MEV Boost

Any suggestions how to resolve the `suitable=0` issue?

----

I may take some guesses:

1. Do we need to provide some seed nodes? Where can we get them from?

2. Is this message from the consensus logs relevant?

   ```
   level=info msg="--weak-subjectivity-checkpoint not provided. Prysm recommends providing a weak subjectivity checkpointfor nodes synced from genesis, or manual verification of block and state roots for checkpoint sync nodes." prefix=blockchain
   ```

3. Perhaps I need to enable "CI Rapid Sync" or "MEV Boost"?

4. Could it be a docker networking issue? (I might try `--network=host` tomorrow. It has helped me many times in the past!)

**Comment by yorickdowne** [comment]
Do an `./ethd update` and try again please. Prysm should fetch a genesis file and show that it did so, on startup.

**Comment by joeytwiddle** [comment]
Hehe, I also tracked it down to the same issue.

Fixed by 1e85ac3e67ad03c807df011ea9b85b1b36e4914f

Thanks!
